crm_claim/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,6 @@
+To configure this module, you need to:
```

Comment:
Please, comment that developer mode is required to view these menus.

Reply:
Done